contrib/.env.sample

```diff
@@ -1,18 +1,26 @@
-ALLOWED_HOSTS=localhost,127.0.0.1,0.0.0.0
-DEBUG=False
-SECRET_KEY=jaa-aaa-ar-bas
+ALLOWED_HOSTS=*
+DEBUG=True
```

Comment:
I tend to prefer `DEBUG=False` as default for security reasons: if someone accidentally deploy with the default settings, there's no risk of exposing too much about the system in a production environment.

Reply:
> The rest of the values have defaults for a good development environment 

I got it. It's a valid claim. But my concern was related to security, not to an easy way to get things going for a development environment.

Just ponder it and I'll be fine with whatever decision you make : )